From 4f30baf38df68e47f2e993b40ebc817435e18d08 Mon Sep 17 00:00:00 2001
From: Rinigus <rinigus.git@gmail.com>
Date: Tue, 22 Sep 2020 22:25:54 +0300
Subject: [PATCH 1/5] Use GLESv2

---
 qt5-qtwebengine.spec | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/qt5-qtwebengine.spec b/qt5-qtwebengine.spec
index 94ffa85..0e63dce 100644
--- a/qt5-qtwebengine.spec
+++ b/qt5-qtwebengine.spec
@@ -142,7 +142,8 @@ BuildRequires: pkgconfig(gobject-2.0)
 BuildRequires: pkgconfig(glib-2.0)
 BuildRequires: pkgconfig(fontconfig)
 BuildRequires: pkgconfig(freetype2)
-BuildRequires: pkgconfig(gl)
+#BuildRequires: pkgconfig(gl)
+BuildRequires: pkgconfig(glesv2)
 BuildRequires: pkgconfig(egl)
 BuildRequires: pkgconfig(jsoncpp)
 BuildRequires: pkgconfig(libpng)
-- 
2.26.2

