From 7d10a1adc73ffc3af2f026b4b0ec50aba53784f9 Mon Sep 17 00:00:00 2001
From: Rinigus <rinigus.git@gmail.com>
Date: Wed, 23 Sep 2020 20:39:28 +0300
Subject: [PATCH 5/5] disable parallel build to fit OBS nodes

---
 qt5-qtwebengine.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/qt5-qtwebengine.spec b/qt5-qtwebengine.spec
index f5a9886..f842259 100644
--- a/qt5-qtwebengine.spec
+++ b/qt5-qtwebengine.spec
@@ -442,7 +442,7 @@ cp -p src/3rdparty/chromium/LICENSE LICENSE.Chromium
 
 %build
 export STRIP=strip
-export NINJAFLAGS="%{__ninja_common_opts} %{?_smp_mflags}"
+export NINJAFLAGS="%{__ninja_common_opts} -j1" #%{?_smp_mflags}"
 export NINJA_PATH=%{__ninja}
 
 %{qmake_qt5} \
@@ -453,7 +453,7 @@ export NINJA_PATH=%{__ninja}
   .
 
 # avoid %%make_build for now, the -O flag buffers output from intermediate build steps done via ninja
-make %{?_smp_mflags}
+make -j1 #%{?_smp_mflags}
 
 %if 0%{?docs}
 %make_build docs
-- 
2.26.2

